src/components/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./Sidebar.css";
+import Home from "../components/icons/home";
+import Menu from "../components/icons/menu";
+import Down from "../components/icons/down";
+
+interface SidebarProps {
+  setTitle: (title: string) => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ setTitle }) => {
+  const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+  return (
+    <div className="sidebar">
+      <div className="menu-item" onClick={() => setTitle("Start")}>
```

Comment:
You should avoid using lamba functions directly here in callback. You should define these callback as constants wrapping them with 'useCallback' hook to cache them. Or even declare them out of the component. You do not want that browser declare them and initialize each time component re-renders, it's time consuming. Check this article:

https://jonhilton.net/typescript-and-react-forbidden-lambdas/

Please check all places in code where you have it defined like this.